Makefile

```diff
@@ -25,6 +27,8 @@ RESOURCES := $(wildcard src/res/*)
 JAVA_HELPER_CLASSES := $(wildcard src/helper/one/profiler/*.class)
 API_SOURCES := $(wildcard src/api/one/profiler/*.java)
 CONVERTER_SOURCES := $(shell find src/converter -name '*.java')
+TEST_SOURCES := $(shell find test -name '*.java')
+TESTS=$(shell ls test/test)
```

Comment:
When possible, prefer [Makefile built-in functions](https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html) to shell commands.